### ISSUE TITLE
`throw null` in `Element.get()`

### ISSUE THREAD

**Issue by DmitrySharabin** [madatajs/madata#29]
Is there a reason to `throw null` instead of `return null` here? I'm wondering because in the console it looks like something bad happens when there is no data in `localStorage` yet. I noticed it when working with `<ma-data>` Vue 3 component. Maybe it's just me. 🤷🏻‍♂️

https://github.com/madatajs/madata/blob/2e8b048cf36b734d2f30caa9b0d1acd799e22811/backends/basic/local.js#L23

**Comment by LeaVerou** [comment]
Do other backends throw if the data is not found? If so, probably for that. If not, we should change it.

**Comment by DmitrySharabin** [comment]
I looked through the code and found that in most cases (`Backend`, `Dropbox`, `GithubGist`, `GithubFile`, `GoogleDrive`) when there is no data to work with, backends return `null`.

However, I noticed inconsistencies with other backends: `Google Calendar`, `Google Firebase`, and `Google Sheets` should properly handle `404` errors and not throw an exception in those cases.

So it seems that `Element.get()` should return `null` and not throw it.
